Reloj_checador/app/views/home/index.html.erb

```diff
@@ -6,7 +6,7 @@
     </head>
 
     <body>
-    <div class="container">
+    <div class="container container2">
```

Comment:
Does this element needs these two CSS classes? I think it would be fine with only one of them. 🤔

Reply:
Deleted the container class and changed container2 with login-container class.